src/Activities/Option/ActivityOption.cs

```diff
@@ -1,16 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-namespace Microsoft.Omex.Extensions.Abstractions.Option
+namespace Microsoft.Omex.Extensions.Activities.Option
 {
 	/// <summary>
 	/// Monitoring option
 	/// </summary>
-	public class MonitoringOption
+	public class ActivityOption
 	{
 		/// <summary>
 		/// Path to the setting
 		/// </summary>
 		public static string MonitoringPath = "Monitoring";
+
+		/// <summary>
+		/// Disable ActivityEventSender so Activity metric is only sent via ActivityMetricsSender
+		/// </summary>
+		public bool DisableActivityEventSender { get; set; }
```

Comment:
```suggestion
		public bool Enabled { get; set; } = true;
```
This will follow the convention that we have for config providers.

Reply:
The answer for this and the above comments is: I want to have all settings are inside "Monitoring" parent, like this 
<img width="518" alt="image" src="https://github.com/microsoft/Omex/assets/17307105/4a73bc40-9e65-49da-9dc4-1ad82274de1f">

Reply:
Would be better to name it ActivityEventSenderEnabled or ActivityEventSourceEnabled or something.
Enabled ties in with convention elsewhere and improves readability, while ActivityEventSource rather than ActivityEventSender would make more sense in context of the service that it is configured.
As a service owner, I want to disable event source emission. I don't want to have to know the specific class name in the abstraction layers.